### PR TITLE
list reproducible artifacts

### PR DESCRIPTION
**Pull request by aTonyXiao**

### CODE REVIEW

File: bugswarm/common/rest_api/database_api.py [hunk 1]
```diff
@@ -83,7 +83,9 @@ def find_artifact(self, image_tag: str, error_if_not_found: bool = True) -> Resp
         return self._get(DatabaseAPI._artifact_image_tag_endpoint(image_tag), error_if_not_found)
 
     def list_artifacts(self) -> List:
-        return self._list(DatabaseAPI._artifacts_endpoint())
+        # return self._list(DatabaseAPI._artifacts_endpoint())
```
Comment (dtomassi) [inline bugswarm/common/rest_api/database_api.py]: Do we want the commented line?

Comment (aTonyXiao) [reply bugswarm/common/rest_api/database_api.py]: @dtomassi We might want to keep it for future reference. What do you think?

Comment (dtomassi) [reply bugswarm/common/rest_api/database_api.py]: I think this PR is fine enough documentation.

File: bugswarm/common/rest_api/database_api.py [hunk 2]
```diff
@@ -79,13 +79,28 @@ def insert_artifact(self, artifact) -> Response:
         return self._insert(DatabaseAPI._artifacts_endpoint(), artifact, 'artifact')
 
     def find_artifact(self, image_tag: str, error_if_not_found: bool = True) -> Response:
+        """
```
Comment (dtomassi) [inline bugswarm/common/rest_api/database_api.py]: Can we add the example?

Comment (aTonyXiao) [reply bugswarm/common/rest_api/database_api.py]: Done

**Comment by dtomassi** [comment]
@aTonyXiao Did you see my comment? https://github.com/BugSwarm/common/pull/19#pullrequestreview-216921584

File: bugswarm/common/rest_api/database_api.py (hunk 2 above)
Comment (aTonyXiao) [reply bugswarm/common/rest_api/database_api.py]: Is that should be added to the README?

**Pull request merged**